gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Dec 14 22:58:07 CET 2015
+#Fri Nov 11 11:39:09 PST 2016
```

Comment:
Update: this is indeed something that is supposed to be checked in with the wrapper. It looks like we had an attempt to include the wrapper before that didn't quite take.